src/disco/components/DiscoPane/index.js

```diff
@@ -81,9 +83,15 @@ export class DiscoPaneBase extends React.Component {
   componentDidMount() {
     const {
       _addChangeListeners,
+      _config,
       handleGlobalEvent,
       mozAddonManager,
     } = this.props;
+
+    if (_config.get('server')) {
```

Comment:
I'm just curious what issue is this fixing...

Reply:
it does not fix an issue but make the code more readable.

Reply:
it is related to your previous question/my previous answer.

Reply:
is this even needed tho? I thought componentDidMount lifecycle was only called on client?